### DISCUSSION TITLE
`apis/argoproj.io/v1alpha1` in error, can I restart Argo?

### DISCUSSION THREAD

**Discussion by JDHawken** [argoproj/argo-workflows#12837] [Q&A] [answered]
Hello!
I need some feedback about one error I encounter.
Actually we cannot create any workflow because log in argoworkflow return : 
```
apis/argoproj.io/v1alpha1 dial tcp 10.40.0.1:443: connect: connection refused
```

My question is : How to deal with it?
Firstly, I want to restart Argoworkflow but how much does it cost ? Will I lose actual workflow who running ? 
Thanks !

_edit by agilgur5_: Original [Slack thread](https://cloud-native.slack.com/archives/C01QW9QSSSK/p1711106789796439)

**Comment by shuangkun** [comment, marked as answer]
Is your cluster under stress? Restart workflow-controller won't miss workflows.

**Reply by agilgur5** [reply]
> because log in argoworkflow return :

Are you trying to login to the UI of the Server?

The Server can similarly be restarted whenever; the Server does not process Workflows and can be run HA with multiple replicas.

**Reply by JDHawken** [reply]
Hello guys, sorry for the delay... I complety forgot about it :(.
After the message of @shuangkun, I rollout my deployment and error message in log has gone!

@agilgur5 No problem with the UI interface, it's was connection issue between ArgoWorkFlow and a HIM developed by us, the HIM need API to get some data from workflows (If I understand well :D )
